src/js/plugins/networkcontrol.js

```diff
@@ -96,8 +98,8 @@ class NetworkControl extends Plugin {
       let requestButton = document.getElementById('request');
       request.onclick = this.request.bind(this);
 
-      let assignButton = document.getElementById('assign');
-      assignButton.onclick = this.assign.bind(this);
+      //let assignButton = document.getElementById('assign');
+      //assignButton.onclick = this.assign.bind(this);
```

Comment:
same: intended comment? Or forgot to remove ? :)

Reply:
Nope, intentionally left out because the assign doesn't do much. Once we provide a way to actually input the configuration of each interface the assign will come back.